molecule/commands.py

```diff
@@ -159,6 +159,8 @@ def converge(self, idempotent=False, create_instances=True, create_inventory=Tru
             kwargs.pop('_err', None)
             kwargs['_env']['ANSIBLE_NOCOLOR'] = 'true'
             kwargs['_env']['ANSIBLE_FORCE_COLOR'] = 'false'
+            kwargs['_env']['ANSIBLE_CALLBACK_PLUGINS'] = os.path.join(
+                sys.prefix, 'share/molecule/ansible/plugins/callback/idempotence')
```

Comment:
Although it's a great idea to use callback plugin, I'm agree with @retr0h. [kitchen-ansiblepush](https://github.com/ahelal/kitchen-ansiblepush) uses [something similar](https://github.com/ahelal/kitchen-ansiblepush/blob/master/lib/kitchen/provisioner/ansible_push.rb#L222) for testing idempotency, and they clearly point to that this might cause conflict.

Perhaps the better method is to not use environment variable, but instead add `callback_plugins` to `ansible.cfg.j2` template and allow user to override it in some way?